internal/rukpak/preflights/crdupgradesafety/crdupgradesafety_test.go

```diff
@@ -71,8 +73,140 @@ func getManifestString(t *testing.T, crdFile string) string {
 // TestInstall exists only for completeness as Install() is currently a no-op. It can be used as
 // a template for real tests in the future if the func is implemented.
 func TestInstall(t *testing.T) {
-	preflight := newMockPreflight(nil, nil, nil)
-	require.Nil(t, preflight.Install(context.Background(), nil))
+	tests := []struct {
+		name          string
+		oldCrdPath    string
```

Comment:
Based on my comment above about `return nil` vs `continue`, it seems like we may want to test scenarios where there are multiple CRDs at play.

Reply:
In your opinion, is this a blocker for this PR? I do agree we should do this, but wonder if it would be better suited to have as a follow-up. I don't mind doing it, but I'm not sure holding this bug fix on a, not insignificant, unit test refactor is worth it.

Reply:
Not a blocker!